gramps/gui/views/treemodels/eventmodel.py

```diff
@@ -145,7 +145,7 @@ def column_place(self, data):
             return ""
 
     def column_type(self, data):
-        return str(EventType(data["type"]))
+        return EventType.get_str(data["type"])
```

Comment:
Why does it have to be `EventType.get_str()` rather than `GrampsType.get_str()`? Is it because each type has a `_CUSTOM`? But isn't it always the same value?

Reply:
I believe _CUSTOM is common for all types.
`GrampsType.__get_str`, which this call ultimately ends up in, needs to types `_I2SMAP`. `_I2SMAP` differs for each type.